boards/riscv/longan_nano/longan_nano_lite.yaml

```diff
@@ -8,6 +8,7 @@ toolchain:
 flash: 64
 ram: 20
 supported:
+  - adc
```

Comment:
I think this yaml file is used for `tests`, and there have a CI failure for `adc_api` test.

Please remove `adc` support from this file or add related dts overlay to `zephyr/tests/drivers/adc/adc_api/boards`.

Reply:
Thank you for looking into that. I will update it as per your comments.